magnetis/astro-native
# feat(TabsContainer): update TabsContainer to receive active and inactive as default props and change internal component [CONFIA-679]

# What
Updates TabsContainer with active and inactive default props and changes internal components and stylizations. 


# Why
Following the new needs to optimize our components.


# How

1. Utilizing `active` and `inactive` default props to `TabsContainer`;
2. Replacing `Tab` component with `Button` component and its properties;
3. Updating `TabsContainer` storybook to receive the new properties;
4. Updating tests and test related properties in `Button` and `TabsContainer` components.



# Sample

iOS:

![Screenshot 2023-01-11 at 15 20 49](https://user-images.githubusercontent.com/35939685/211886783-546a4828-0d93-4c91-8f8f-5b98d7838fdc.png)

Android:

![Screenshot 2023-01-11 at 15 38 54](https://user-images.githubusercontent.com/35939685/211890432-b0a54a8e-dc23-4798-bf9a-dbc906db8d18.png)





# QA

1. Start storybook with `yarn storybook`;
2. Run your preferred simulator;
4. In the storybook page, choose `TabsContainer` in `Tabs`;
5. See if `rounded`, `ActiveType`, `ActiveVariant`, `InactiveType` and `InactiveVariant` props are working correctly;








[CONFIA-679](https://produtomagnetis.atlassian.net/browse/CONFIA-679)

[CONFIA-679]: https://produtomagnetis.atlassian.net/browse/CONFIA-679?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
